src/wp-includes/rest-api.php

```diff
@@ -1258,12 +1260,48 @@ function rest_validate_value_from_schema( $value, $args, $param = '' ) {
 			return new WP_Error( 'rest_invalid_param', sprintf( __( '%1$s is not of type %2$s.' ), $param, 'array' ) );
 		}
 
+		if ( isset( $args['minItems'] ) ) {
+			if ( $args['minItems'] < 0 ) {
```

Comment:
We can skip the checks that the schema itself is correct. At some point it'd be a good idea to validate the schema separately, but I don't think we should do that piecemeal and instead think out a solid plan for it. We don't apply that validation in the number constraints for instance.

Reply:
Ok